content/id/docs/concepts/policy/limit-range.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Batas Rentang (Limit Range)
```

Comment:
LimitRange is an API object, let's just use LimitRange without translation as title?
we can add "Batas Rentang" in the beginning of paragraph to add meaning to it

Reply:
Fixed.